pkg/config/config.go

```diff
@@ -260,7 +261,7 @@ func Load() {
 
 	if len(clowder.KafkaServers) > 0 {
 		log.Warn().Msgf("clowder.KafkaServers has length of %s", strconv.Itoa(len(clowder.KafkaServers)))
-		LoadedConfig.NotificationsClient = SetupNotifications(clowder.KafkaServers)
+		LoadedConfig.NotificationsClient = SetupNotifications(clowder.KafkaServers, LoadedConfig.Kafka.Capath)
```

Comment:
i don't think this is quite right, we don't load the clowder() kafka config into our own.  i think you need to pull it from clowder: 

``` ca, err := clowder.LoadedConfig.KafkaCa(clowder.LoadedConfig.Kafka.Brokers...)```

you'll need to wrap it in a clowder.IsClowderEnabled()  too

Reply:
actually, i'm wrong, but lets log the Capath while we're at it?